app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -139,6 +148,7 @@ class BrowserFragment :
         return binding.root
     }
 
+    @SuppressLint("VisibleForTests")
```

Comment:
Is there any specific reason why we need to suppress? :)

Reply:
Because DownloadsUseCases from DownloadsFeature constructor is only VisibleForTests . It's the same problem like in Fenix .